pyproject.toml

```diff
@@ -1,3 +1,12 @@
+[project]
+name = "data-architecture"
+version = "0.0.1"
+readme = "README.md"
+requires-python = ">=3.11"
+
+[tool.pyright]
+reportUnusedImport = false
```

Comment:
Adding this to the global tool config since `ruff` already handles checking for unused imports.